website/docs/guides/using-terraform-templates.mdx

```diff
@@ -92,3 +93,282 @@ gitops get template terraform tf-controller-aurora --list-parameters --endpoint
+
+```yaml title="./clusters/<cluster-name>/sample-rds-tf-template.yaml"
+---
+apiVersion: clustertemplates.weave.works/v1alpha1
+kind: GitOpsTemplate
+metadata:
+  name: sample-aurora-tf-template
+  namespace: default
+spec:
+  description: This is a sample Aurora RDS template.
+  params:
+    - name: RESOURCE_NAME
+      description: Resource Name
+    - name: CLUSTER_IDENTIFIER
+      description: Cluster Identifier
+    - name: DATABASE_NAME
+      description: Database Name
+    - name: BACKUP_RETENTION_PERIOD
+      description: Backup Retention Period
+    - name: REGION
+      description: Region
+    - name: S3_IMPORT_BUCKET
+      description: S3 Import Bucket
+  resourcetemplates:
+    - apiVersion: infra.contrib.fluxcd.io/v1alpha1
+      kind: Terraform
+      metadata:
+        name: ${RESOURCE_NAME}
+        namespace: flux-system
+      spec:
+        interval: 1h
+        path: ./
+        approvePlan: auto
+        alwaysCleanupRunnerPod: true
+        vars:
+        - name: cluster_identifier
+          value: ${CLUSTER_IDENTIFIER}
+        - name: database_name
+          value: ${DATABASE_NAME}
+        - name: backup_retention_period
+          value: ${BACKUP_RETENTION_PERIOD}
+        - name: region
+          value: ${REGION}
+        - name: s3_import_bucket
+          value: ${S3_IMPORT_BUCKET}
+        varsFrom:
+        - kind: Secret
+          name: tf-controller-auth
+        sourceRef:
+          kind: GitRepository
+          name: flux-system
+          namespace: flux-system
+```
+
+Add the following Terraform manifest to the root of your Git repository.
+
+```yaml title="./rds.tf"
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+variable "cluster_identifier" {}
+variable "database_name" {}
+variable "master_username" {}
+variable "master_password" {}
+variable "backup_retention_period" {}
+variable "region" {}
+variable "s3_import_bucket" {}
+variable "aws_access_key" {}
+variable "aws_secret_key" {}
+variable "aws_role_arn" {}
+
+provider "aws" {
+  region = var.region
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+
+  assume_role {
+    role_arn    = var.aws_role_arn
+  }
+}
+
+locals {
+  engine         = "aurora-mysql"
+  engine_version = "5.7.mysql_aurora.2.07.5"
+  port           = 3306
+}
+
+resource "aws_s3_bucket" "s3" {
+  bucket = var.s3_import_bucket
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_public_access_block" "s3" {
+  bucket = aws_s3_bucket.s3.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+resource "aws_s3_bucket_object" "init_sql" {
+  bucket  = aws_s3_bucket.s3.id
+  key     = "init.sql.tar.gz"
+  content = <<EOM
+CREATE DATABASE test;
```

Comment:
I got the syntax for creating a table wrong :p
it should be, e.g.:
```sql
CREATE DATABASE testdb;
CREATE TABLE test(
   id     INT PRIMARY KEY  NOT NULL,
   value  CHAR(50)         NOT NULL
);
```

Reply:
thanks Sam!! resolving this since I removed the S3 bucket migration like we discussed :)